package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "dex-react",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "NODE_ENV=development webpack-dashboard -- webpack-dev-server --mode development --hot",
+    "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
+    "eslint": "eslint . --ext .ts,.tsx,.js",
+    "eslint-fix": "eslint --fix . --ext .ts,.tsx,.js",
+    "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist"
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
I think we said the other day for now we are not sure about the licenses. So we better remove them. Otherwise ask chris. Anyways, I think MIT is not for sure.